packages/react-day-picker/src/types/DayPickerDefault.ts

```diff
@@ -1,10 +1,10 @@
 import { DayPickerProps } from 'DayPicker';
 
-import { DayPickerBase, DaySelectionMode } from './DayPickerBase';
+import { DayPickerBase } from './DayPickerBase';
 
 /** The props for the {@link DayPicker} component when using `mode="default"` or `undefined`. */
 export interface DayPickerDefaultProps extends DayPickerBase {
-  mode?: DaySelectionMode;
+  mode?: undefined | 'default';
```

Comment:
You already have `?` near mode so you don’t need to provide explicit `undefined`:

`mode?: 'default'`

Isn’t it?

Reply:
Actually there's a setting in tsconfig where this would be required (I couldn't find which right now), so I'd better keep it explicit.

Reply:
Found the option: https://www.typescriptlang.org/tsconfig/exactOptionalPropertyTypes.html

Reply:
![image](https://user-images.githubusercontent.com/2149612/193488050-ebff0bdf-3841-4f22-b0e0-4297f1ff650b.png)

FYI regarding `?` and unsetting it :)